src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.hpp

```diff
@@ -269,7 +269,8 @@ class KerrSchild : public AnalyticSolution<3_st>,
   void pup(PUP::er& p);
 
   SPECTRE_ALWAYS_INLINE double mass() const { return mass_; }
-  SPECTRE_ALWAYS_INLINE const std::array<double, volume_dim>& center() const {
+  SPECTRE_ALWAYS_INLINE const std::array<double, volume_dim>& expansion_center()
```

Comment:
I think this should be called physical_center, as it is independent of a spherical harmonic expansion

Reply:
what is "physical" though :) I never know what this word means when I read it..

Reply:
do you have a better name?

Reply:
there are no coefficients in the solution...

Reply:
Would it be so bad for KerrSchild to still call it `center()` since for KerrSchild there is no ambiguity?

Reply:
I would strongly prefer `center` over `expansion_center` and `dipole_center`

Reply:
Ok yeah I can see how for KerrSchild `center` would be better since they are all the same. Any others I should change?

Reply:
I'm also not sure about `BoundaryConditions/ApparentHorizon.hpp`.   If there's no ambiguity I would keep it `center`. If there is an ambiguity (like in Strahlkorper) then I would give it a more descriptive name.

Reply:
@nilsleiffischer What do you think about the elliptic system BC ApparentHorizon.hpp?

Reply:
I agree with @markscheel that I'd prefer to change only names that are ambiguous